.github/workflows/link-checker.yml

```diff
@@ -4,9 +4,6 @@ on:
   # Check links once a week on Monday
   schedule:
     - cron: "0 0 * * 1"
-  pull_request:
-    branches:
-      - "master"
```

Comment:
We could move this to a flowzone custom test action, but why disable it now?

Reply:
Not being disabled. I just felt it was not helping a lot when it was running as part of a PR. Let me do that in another PR

Reply:
Yeah, I don't think we need to disable that as part of this PR.

Reply:
@klutchell Already had restored the changes to this file as per my last comment. Forgot to resolve this. Please resolve this comment when you see it.